frontend/app/routes/$lang/_public/apply/$id/adult-child/children/$childId/information.tsx

```diff
@@ -266,10 +266,8 @@ export default function ApplyFlowChildInformation() {
           <InputField
             id="social-insurance-number"
             name="socialInsuranceNumber"
-            label={t('apply-adult-child:children.information.sin')}
+            label={t('apply-adult-child:children.information.help-message.sin')}
```

Comment:
**Nitpick:** Consider changing the value of `apply-adult-child:children.information.sin` rather than using the `help-message` key as the label, which is a bit misleading IMO.

We can then remove the `help-message` i18n key if it is no longer used.

Reply:
We can revert this change since `apply-adult-child:children.information.help-message.sin` has been removed.

Reply:
Wait sorry, just noticed that the change wasn't made in `apply-adult-child.json`.